codetools/contexts/tests/data_context_test_case.py

```diff
@@ -41,6 +59,18 @@ def test_persistence():
     assert set(d2.keys()) == set(['a', 'b'])
     assert d2['a'] == d['a']
     assert d2['b'] == d['b']
+    
+
+def test_long_term_persistence():
```

Comment:
bikeshedding the test name 😆 
```suggestion
def test_persistence_backwards_compatibility():
```